apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -225,3 +253,78 @@ function PopoverEndDate() {
     </p>
   `.toString();
 }
+
+function CreateCourseInstanceModal({
+  csrfToken,
+  initialStartDateFormatted,
+  initialEndDateFormatted,
+}: {
+  csrfToken: string;
+  initialStartDateFormatted?: string;
+  initialEndDateFormatted?: string;
+}) {
+  return Modal({
+    id: 'createCourseInstanceModal',
+    title: 'Create Course Instance',
```

Comment:
Conventionally we use sentence case here:

```suggestion
    title: 'Create course instance',
```